README.md

```diff
@@ -61,28 +61,16 @@ Example `secrets.env.example` and `aws.env.example` files are generated for conv
 files will be required for deployment. The populated files (which will contain secrets) should not be included in this
 repository.
 
-#### Templates
+#### Templates — `/data/apis/{sanitizedApiName}/templates`
 
-`/data/apis/{apiName}/templates`
-
-- Initialised during creation of the boilerplate and is manually populated.
-- Files can be named arbitrarily as they are renamed according to the `templateName` on `validate`.
```

Comment:
These all can potentially be moved into (in-repo) guides to creating templates and Beacons